tests/modules/lib/cddl-gen/decode/CMakeLists.txt

```diff
@@ -0,0 +1,35 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor
```

Comment:
The nit of the month, 2022

Reply:
The PR was opened in 2021, so I guess copyrighting this file the year it was created and published for the first time is the right thing to do.